src/models/__tests__/DataSets.spec.js

```diff
@@ -59,7 +71,7 @@ describe("DataSets", () => {
                     order: "name:idesc",
                     page: 2,
                     pageSize: 10,
-                    filter: ["displayName:ilike:abc", "user.id:eq:b123123123"],
+                    filter: ["displayName:ilike:abc", "user.id:eq:b123123123", "id:in:[]"],
```

Comment:
It would be cool to have another test where listMock returns datasets, some of theme having own attribute->attributeValues to true, and then check that those dataset IDS appears in a "id:in:[ID1,ID2]" filter.